khiva/matrix.py

```diff
@@ -77,6 +77,10 @@ def find_best_n_motifs(profile, index, m, n, self_join=False):
 def stomp(first_time_series, second_time_series, subsequence_length):
     """ Stomp algorithm to calculate the matrix profile between `ta` and `tb` using a subsequence length of `m`.
```

Comment:
Does this method filter trivial matches? if so, please add, "This method filters the trivial matches.", as the one below

Reply:
This method doesn't filter trivial matches as it consider that both time series are different.